ui/src/app/views/workspace-list.tsx

```diff
@@ -79,26 +79,27 @@ const WorkspaceCardMenu: React.FunctionComponent<{
       side='bottom'
       closeOnClick
       content={ <React.Fragment>
-        <MenuItem onClick={() => {navigate([wsPathPrefix, 'duplicate']); }}>
+        <MenuItem icon='copy'
+                  onClick={() => {navigate([wsPathPrefix, 'duplicate']); }}>
           Duplicate
         </MenuItem>
         <TooltipTrigger content={<div>Requires Write Permission</div>}
                         disabled={wp.canWrite}>
-          <MenuItem data-test-id='edit-workspace'
```

Comment:
these `data-test-id`s are not necessary since MenuItem automatically creates one